tests/language-feature/capability/capability-inheritance-1.slang

```diff
@@ -0,0 +1,41 @@
+//TEST:SIMPLE(filecheck=CHECK): -target hlsl -entry computeMain -stage compute -DDISJOINT
+//TEST:SIMPLE(filecheck=CHECK_CHILD_FAIL): -target hlsl -entry computeMain -stage compute -DCHILD_IS_SUPERSET_FAIL
+//TEST:SIMPLE(filecheck=CHECK_CHILD_PASS): -target hlsl -entry computeMain -stage compute -DCHILD_IS_SUPERSET_PASS
+//TEST:SIMPLE(filecheck=CHECK_STAGE): -target hlsl -entry computeMain -stage compute -DSTAGE_MISMATCH
+
+// Test if InheritanceDecl fail to conform to parent capabilities
```

Comment:
Can you rewrite this test to make it more readable. Don't use #defines, just declare a different set of interface and impl types with comments explaining what they are testing.